newIDE/app/src/EventsSheet/index.js

```diff
@@ -363,6 +368,20 @@ export default class EventsSheet extends React.Component<Props, State> {
     return newEvents;
   };
 
+  //TODO ideally see for use toggle
+  openTextEditorDialog = () => {
+    this.setState({
+      textEditorDialog: true,
+    });
+  };
+
+  //TODO close and save in history
+  closeTextEditorDialog = () => {
+    this.setState({
+      textEditorDialog: false,
```

Comment:
As per my previous comment, this can be instead `textEditedEvent: null`. Conceptually, this mean "We don't have anymore an event that we're editing text for.".

Reply:
Fixed